services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py

```diff
@@ -192,8 +189,6 @@ async def store_to_s3(  # pylint:disable=too-many-locals,too-many-arguments
         f"{source_dir}",
         f"dst:{s3_path}",
         "--verbose",
-        "--multi-thread-streams",
```

Comment:
did you not try to remove the buffer-size and use-mmap to see if ti gets any increments compare to the was CLI tools?

Reply:
yes I did, please check the discussion linked above. it makes 15 seconds difference, aka no changes at all.
Sorry but this tool just does not compare well with aws CLI.